### ISSUE TITLE
Running Issue: Caltrans sprint outcomes/themes

### ISSUE THREAD

**Issue by jasonlally** [cagov/caldata-mdsa-caltrans-pems#99]
**Sprint 2024-013 Goals:**

- [x] #170
- [x] #186
- [ ] #255 
- [ ] #256
- [x] #195
- [x] #115 (depends on #114 )
- [ ]  #185
- [ ] #211 
- [ ] #233 
- [ ] #226 

Stretch goals:
- [ ] #70 Start exploring how to load LCS / CHP data
- [ ] #71 lower priority after VDS metadata
- [ ] Advanced Metrics:
     - [ ] #187 waiting on #186
- [ ] #204
- [ ] #103

**Comment by jkarpen** [comment]
Note: Per discussion on Slack, changing this from a single-sprint issue to a running issue so it can be bookmarked and referenced more easily by the team. Each Sprint Planning we will move the prior sprint's goals down into a comment, then add the new sprint goals to the description at the top.

**Comment by jkarpen** [comment]
**Sprint 2024-05 Goals:**

- [ ] 1. Finish the data relay server such that data is flowing from Caltrans servers to Snowflake (parsing data in future sprint)
- [ ] 2. Getting 2 or 3 of the detector health diagnostic tests in (#79, #80, #81, #82 may be possible, at least starting points)
- [x] 3. Deliver working session on incremental models

Stretch goal:
- [ ] 1. Prioritize / begin to prioritize remaining metrics (possibly start with agreement on criteria/how we'll make those decisions)

**Comment by jkarpen** [comment]
**Sprint 2024-06 Goals:**

- [ ] Complete health diagnostic test models #78 , #83, #84 .
- [ ] Convert all diagnostic tests in #85 to incremental models 
- [ ] #114 
- [ ] #115 
- [x] #117
- [ ] #127 

Stretch goals:
- [ ] Start working on imputation on the raw data based on Iteris documentation
- [ ] Prioritize / begin to prioritize remaining metrics (possibly start with agreement on criteria/how we'll make those decisions) #69 
- [x] #120 
- [x] #121

**Comment by jkarpen** [comment]
Sprint 2024-07 Goals:

 Complete health diagnostic test models https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/78 , https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/83, https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/84 (if not finished by end of day 3/29).
 https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/142
 https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/114
 https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/115
 https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/126
 https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/143
Stretch goals:

 Start working on imputation on the raw data based on Iteris documentation
 Possibly start work on advanced metrics (lane closure, etc) - lower likelihood this sprint
 Start exploring how to load LCS / CHP data https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/70 https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/71

**Comment by jkarpen** [comment]
**Sprint 2024-08 Goals:**

- [ ] Complete health diagnostic test models 
   - [x] #83 
   - [ ] #84 (if not finished by end of day 4/12).
- [x] #142 
- [ ] #114 
- [ ] #126 - 
   - [ ] @kengodleskidot working on #154 , #155 , #156 
   - [ ] @ZhenyuZhu-Caltrans  working on #144
   - [ ] @britt-allen working on #153 
- [x] #143 

Stretch goals:
- [ ] #173 
- [ ] Start exploring how to load LCS / CHP data #70 #71 
- [ ] Start work on #152 
- [ ] #115 (depends on #114 )

**Comment by junlee-analytica** [comment]
**Sprint 2024-09 Goals:**

- [x] Complete health diagnostic test models 
   - [x] #84 (if not finished by end of day 4/26).
- [x] #114 
- [ ] #126 - 
   - [x] @britt-allen working on #153 
   - [ ] @kengodleskidot  working on #152 
- [ ] #169 (Ian/Mintu driving)
- [ ] #136 @britt-allen 
- [x] #183 
- [x] #163 @kengodleskidot 

Stretch goals:
- [ ] #115 (depends on #114 )
- [ ] Start exploring how to load LCS / CHP data #70 #71 (Pingping and Ian driving) - lower priority after VDS metadata
- [ ] Advanced Metrics:
     - [ ] #185
     - [ ] #186
     - [ ] #187
- [ ] #182 (Pingping driving, lower priority than the metadata issue above)

**Comment by junlee-analytica** [comment]
**Sprint 2024-010 Goals:**

- [x] #126 - 
   - [x] #153 @britt-allen
   - [x] #152 @kengodleskidot
- [ ] #169 @ian-r-rose & @mmmiah 
- [ ] #170 @mmmiah 
- [x] #136 @britt-allen 
- [ ] #186 @britt-allen 
- [ ] #205 @mmmiah 
- [ ] #195 @pingpingxiu-DOT-ca-gov 
- [ ] #115 (depends on #114 )

Stretch goals:
- [ ] #70 Start exploring how to load LCS / CHP data
- [ ] #71 (Pingping and Ian driving) - lower priority after VDS metadata
- [ ] Advanced Metrics:
     - [ ] #185 @pingpingxiu-DOT-ca-gov 
     - [ ] #187 waiting on #186
- [ ] #182 (Pingping driving, lower priority than the metadata issue above)
- [ ] #204

**Comment by junlee-analytica** [comment]
**Sprint 2024-011 Goals:**

- [ ] #169 @ian-r-rose & @mmmiah 
- [ ] #170 @mmmiah 
- [ ] #186 @britt-allen 
- [ ] #205 @mmmiah 
- [ ] #195 @pingpingxiu-DOT-ca-gov 
- [ ] #115 (depends on #114 )
- [ ]  #185 @pingpingxiu-DOT-ca-gov 
- [x] #140 
- [ ] #217 @kengodleskidot 
- [x] #218 
- [ ] #211 
- [ ] #222 
- [ ] #139 

Stretch goals:
- [ ] #70 Start exploring how to load LCS / CHP data
- [ ] #71 (Pingping and Ian driving) - lower priority after VDS metadata
- [ ] Advanced Metrics:
     - [ ] #187 waiting on #186
- [x] #182 (Pingping driving, lower priority than the metadata issue above)
- [ ] #204 @kengodleskidot @mmmiah 
- [ ] #103

**Comment by junlee-analytica** [comment]
**Sprint 2024-012 Goals:**

- [x] #169 @ian-r-rose & @mmmiah 
- [ ] #170 @mmmiah 
- [ ] #186 @britt-allen 
- [x] #205 @mmmiah 
- [ ] #195 @pingpingxiu-DOT-ca-gov 
- [ ] #115 (depends on #114 )
- [ ]  #185  @mmmiah 
- [x] #217 @kengodleskidot 
- [ ] #211 
- [x] #222 
- [x] #139 

Stretch goals:
- [ ] #70 Start exploring how to load LCS / CHP data
- [ ] #71 (Pingping and Ian driving) - lower priority after VDS metadata
- [ ] Advanced Metrics:
     - [ ] #187 waiting on #186
- [ ] #204 @kengodleskidot @mmmiah 
- [ ] #103

**Comment by jkarpen** [comment]
Backlogging this issue for now since we are switching to the project board for tracking priorities.

**Comment by britt-allen** [comment]
Why are we not closing or deleting the issue?

**Comment by ian-r-rose** [comment]
We can close it for now, and revisit if we find we want it again
